frontend/src/assets/css/scaffolding.scss

```diff
@@ -33,6 +36,28 @@
   --supprot-error-color: #f76519;
 }
 
+// custom scrollbar
+html {
+  scrollbar-color: $brand-green-color $green-100-header-menu-background;
+  scrollbar-width: thin;
+}
```

Comment:
Please read an article how to customize scrollbar https://css-tricks.com/almanac/properties/s/scrollbar/

Reply:
Also please move all scrollbar related styles to separate separate file with mixins. Then apply added mixin here